recipes/clangdev/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "3.8.1" %}
+{% set sha256 = "4cd3836dfb4b88b597e075341cae86d61c63ce3963e45c7fe6a8bf59bb382cdf" %}
+
+package:
+  name: clangdev
```

Comment:
While it is "clang the compiler", the package also contains all the development stuff (headers/libraries) for it, hence the `dev`. The LLVM package is also `llvmdev`, so I feel this naming is consistent.